docs/how_to/visualize_a_quantum_circuit.rst

```diff
@@ -0,0 +1,140 @@
+    :options: +NORMALIZE_WHITESPACE
+
+         ┌───┐     ┌─┐   
+    q_0: ┤ H ├──■──┤M├───
+         ├───┤┌─┴─┐└╥┘┌─┐
+    q_1: ┤ H ├┤ X ├─╫─┤M├
+         ├───┤└┬─┬┘ ║ └╥┘
+    q_2: ┤ H ├─┤M├──╫──╫─
+         └───┘ └╥┘  ║  ║ 
+    c: 3/═══════╩═══╩══╩═
+                2   0  1 
+
+:meth:`~.QuantumCircuit.draw`
+---------------------------------------------
+
+You can also call the :meth:`~.QuantumCircuit.draw` method on a :class:`~.QuantumCircuit` object to visualize it. If you don't specify any arguments you will get a plain text representation of the circuit.
+
+.. code-block:: python
+
+    qc.draw()
+
+.. code-block:: text
+
+         ┌───┐     ┌─┐   
+    q_0: ┤ H ├──■──┤M├───
+         ├───┤┌─┴─┐└╥┘┌─┐
+    q_1: ┤ H ├┤ X ├─╫─┤M├
+         ├───┤└┬─┬┘ ║ └╥┘
+    q_2: ┤ H ├─┤M├──╫──╫─
+         └───┘ └╥┘  ║  ║ 
+    c: 3/═══════╩═══╩══╩═
+                2   0  1  
+
+However, if you change the ``output`` argument, you can get other different renderings. The available options for this argument are:
+
+* ``'text'``: renders the circuit with ASCII art. It's the default option.
+* ``'mpl'``: uses `matplotlib <https://matplotlib.org/>`_ to render the circuit.
+* ``'latex'``: uses :math:`\LaTeX` to render the circuit. It requires a full `LaTeX <https://latex.org/forum/>`_ distribution and the package ``pdflatex``.
+* ``'latex_source'``: outputs the :math:`\LaTeX` source code that creates the ``'latex'`` rendering of the circuit.
+
+Because this optional or keyword argument is actually the first of this method, one can type ``qc.draw(option)`` instead of ``qc.draw(output=option)``.
+
+.. note::
+    By default, the :meth:`~.QuantumCircuit.draw` method returns the rendered image as an object and does not output anything. The exact class returned depends on the output specified: ``'text'`` (the default) returns a ``TextDrawer`` object, ``'mpl'`` returns a `matplotlib.figure.Figure <https://matplotlib.org/stable/api/figure_api.html?highlight=figure#matplotlib.figure.Figure>`_ object, and ``'latex'`` returns a ``PIL.Image`` object. Having the return types enables modifying or directly interacting with the rendered output from the drawers. Jupyter notebooks understand these return types and render them for us in this guide, but when running outside of Jupyter, you do not have this feature automatically. However, the :meth:`~.QuantumCircuit.draw` method has optional arguments to display or save the output. When specified, the ``filename`` kwarg takes a path to which it saves the rendered output. Alternatively, if you're using the ``'mpl'`` or ``'latex'`` outputs, you can leverage the ``interactive`` kwarg to open the image in a new window (this will not always work from within a notebook).
+
+
+``'mpl'``
+^^^^^^^^^
+
+.. code-block:: python
+
+    qc.draw('mpl')
+
+.. plot::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(3, 3)
+    qc.h(range(3))
+    qc.cx(0, 1)
+    qc.measure(range(3), range(3))
+    qc.draw("mpl")
+
+
```

Comment:
`latex` output style is missing here. I assume it's because CI doesn't have latex installed and can't generate the image. Am I right?

Reply:
Yes. I initially included `latex` output but CI couldn't generate the image.